### ISSUE TITLE
why [2::3]?

### ISSUE THREAD

**Issue by ucas-hao** [SAI990323/TALLRec#65]
可以问一下为什么return logits[:, 1][2::3], gold[2::3]嘛？是因为这样evaluation更加方便快捷嘛？实际也可以return logits[:, 1], gold？
